src/main/java/mate/academy/spring/model/RoleName.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.spring.model;
+
+public enum RoleName {
+    USER,
+    ADMIN
+}
```

Comment:
<img width="600" alt="image" src="https://user-images.githubusercontent.com/58032907/209639449-3bdfc637-66bd-4844-8d59-2e6726e05863.png">

Reply:
I mean that `RoleName` enum declaration can be inside `Role` class
and we can name it like `Type`, for example